shell/components/KeyValueView.vue

```diff
@@ -70,6 +66,14 @@ export default {
   },
 
   computed: {
+    _keyLabel() {
```

Comment:
With the `_` prefix are you trying to denote private scope?

Stylistically I don't think I've run into this in our codebase.

Reply:
Kind of. 
I just had to pair an exposed value to an internal one, so I tried to use the closest naming convention to this case.
I am aware that's usually denotes more of a private scope, so if you think is confusing I'm fine to change it. 
What would you use in that case?

Reply:
I've encountered the convention of using `_` to denote "private" scope in some of our models and classes, for example:

https://github.com/rancher/dashboard/blob/75c595f97f8cfe1d0c2a37242306863ec4d61486/shell/plugins/dashboard-store/resource-class.js#L1709-L1739

I know that I have encountered it elsewhere throughout Dashboard, and it's a common enough convention in javascript where I think that its meaning is clear.

Reply:
It was just a question. It's fine to ship as is.